src/traces/sankey/plot.js

```diff
@@ -73,12 +73,14 @@ function linkHoveredStyle(d, sankey, visitNodes, sankeyLink) {
     var label = sankeyLink.datum().link.label;
 
     sankeyLink.style('fill-opacity', 0.4);
+    sankeyLink.style('stroke-opacity', 0.4);
```

Comment:
Why do we need this now?

Reply:
Actually we won't need to touch `stroke-opacity` on hover anymore if we render links as closed paths instead of open paths with `stroke-width`

This file will end up more similar to the original one!  :tada:

Reply:
Removed in d1d852ab3a4270d3b7a847ba84b6b0ae5e43e967